ds-collector/ds-collector

```diff
@@ -225,12 +231,18 @@ node_connect_ssh() {
 
 bastion_checks() {
   # pre-conditions
+  mkdir -p $baseDir
+  if [ "$?" != "0" ]; then
```

Comment:
If the directory already exists, will we fail out from the mkdir returning non-zero?